docs/topics/rn-resolved-issues.adoc

```diff
@@ -16,15 +16,15 @@ For a complete list of all issues resolved in this release, see the list of link
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-3082[WINDUP-3082]
-|Eclipse plugin
-|A `NullPointerException` error occurs when browsing rules in the Rulesets view.
+|link:https://issues.redhat.com/browse/WINDUP-3125[WINDUP-3125]
+|ItelliJ IDEA plugin and Visual Studio Code extension
+|The *Apply QuickFix* menu option is still active after a QuickFix is applied.
 
-|link:https://issues.redhat.com/browse/WINDUP-3083[WINDUP-3083]
-|Eclipse plugin
-|When you click or double-click an issue in the Issue Explorer pane, the Issue Details pane is blank.
+|link:https://issues.redhat.com/browse/WINDUP-3134[WINDUP-3134]
+|ItelliJ IDEA plugin
+|After you run an analysis, the *Mark as Complete* menu option is not displayed.
 
-|link:https://issues.redhat.com/browse/WINDUP-3084[WINDUP-3084]
+|link:https://issues.redhat.com/browse/WINDUP-3136[WINDUP-3136]
 |Visual Studio Code extension
-|The Open Report button on the Issue Details pane does not work.
+|if you set _JAVA_HOME_ incorrectly and then set it correctly, the *Run* option is deactivated.
```

Comment:
This is missing the step about how the analysis started running when the variable was incorrect. Also, sentences begin with capital letters.

Suggest: If you run an analysis with an incorrect `JAVA_HOME` environmental variable, the analysis continues to run after you correct the variable and the *Run* option is disabled.

Reply:
AFAIK, JAVA_HOME is italicized throughout the docs. I think it's also the only environmental variable I've touched. Should I change the formatting of JAVA_HOME here (and globally in a separate PR)?